api/src/main/java/marquez/db/LineageDao.java

```diff
@@ -79,6 +93,39 @@ SELECT DISTINCT ON (j.uuid) j.*, inputs AS input_uuids, outputs AS output_uuids
   """)
   Set<JobData> getLineage(@BindList Set<UUID> jobIds, int depth);
 
+  /**
+   * 1 level of lineage for all the children jobs of the given parent
+   *
+   * @param parentJobNamespace the namespace of the parent
+   * @param parentJobName the name of the parent
+   * @return edges form job to dataset to job
+   */
+  @SqlQuery(
+      """
+      SELECT
+          jobs.namespace_name AS job_namespace, jobs.simple_name AS job_name,
+          jvim.io_type AS io1,
+          d.namespace_name AS ds_namespace, d."name" AS ds_name,
+          jvim2.io_type AS io2,
+          jv2.namespace_name AS job2_namespace, jv2.job_name  AS job2_name,
+          pj.namespace_name AS job2_parent_namespace, pj.simple_name AS job2_parent_name
+      FROM jobs
```

Comment:
I thought we should always make use os `jobs_view` in order to support feature of hidden jobs. 
This would also limit one join I believe.

Reply:
just replace jobs by jobs_view here? What do you mean by "limit one join"?

Reply:
`jobs_view` already has parent job name and namespace: https://github.com/MarquezProject/marquez/blob/5c42dfd8cd0f2c739f493e301838bb3f4b0c2497/api/src/main/resources/marquez/db/migration/R__1_Jobs_view_and_rewrite_function.sql#L22

Reply:
updated